candi/bashible/common-steps/node-group/050_configure_and_start_containerd.sh.tpl

```diff
@@ -194,6 +194,25 @@ fi
 
 bb-sync-file /etc/containerd/config.toml - containerd-config-file-changed <<< "${containerd_toml}"
 
+# TODO remove when the next version is released !!!
+#
+# This is a oneshot job to remove unnecessary blobs (compressed image layers)
+# pulled before changing the `discard_unpacked_layer` option. In the future,
+# it will be removed automatically once they have been pulled and unpacked.
+#
+# Clean content to save disk space
+if [ ! -f /var/lib/bashible/containerd_content_store_is_cleared ] ; then
+  bb-log-info "Removing compressed image layers from containerd's content store"
+
+  ctr -n k8s.io content prune references
+  if [ $? -eq 0 ]; then
+    touch /var/lib/bashible/containerd_content_store_is_cleared
+  else
+    bb-log-warning "Failed to clean up containerd's content store. Skip this for now"
+  fi
+fi
+
+
```

Comment:
I think this should be NodeGroupConfiguration step, not bashible

Reply:
Moved to the node-manager module